### PR TITLE
PIL-685: Fix images on wallet recovery screens not changing with dark theme

### PR DESCRIPTION
**Pull request by wjkg**
Icons changed on the recovery options and biometrics prompt screens.

### CODE REVIEW

**Comment by wjkg** [comment]
Seems the recovery icon looks fine with that method (it could have been a little blurry due to subtle effects on the image, but isn't), so tintColor it is.

File: src/screens/BiometricsPrompt/BiometricsPrompt.js [hunk 1]
```diff
@@ -128,12 +130,16 @@ class BiometricsPrompt extends React.Component<Props> {
...
+    const colors = getThemeColors(this.props.theme);
     return (
       <ContainerWithHeader headerProps={{ centerItems: [{ title: 'Make crypto easy' }] }}>
         <ContentWrapper contentContainerStyle={{ paddingVertical: 20, paddingHorizontal: 30, flexGrow: 1 }}>
           <HeaderText>{`Would you like to use\n${biometryTypeTitle} with your\nwallet?`}</HeaderText>
           <ContentInnerWrapper>
-            <TouchIdImage source={imageSource} />
+            <TouchIdImage
+              source={imageSource}
+              style={{ tintColor: colors.positive }}
```
Comment (graszka22) [inline src/screens/BiometricsPrompt/BiometricsPrompt.js:141]: can we add this style in a styled component? like this? note I haven't tested if that works :p 
```
const TouchIdImage = styled(CachedImage)`
  width: 164px;
  height: 164px;
  tint-color: ${themedColors.positive};
`;
```

File: src/screens/BiometricsPrompt/BiometricsPrompt.js [hunk 2]
```diff
@@ -162,4 +165,4 @@ const mapDispatchToProps = (dispatch: Dispatch): $Shape<Props> => ({
   ),
 });
 
-export default connect(mapStateToProps, mapDispatchToProps)(BiometricsPrompt);
+export default withTheme(connect(mapStateToProps, mapDispatchToProps)(BiometricsPrompt));
```
Comment (graszka22) [inline src/screens/BiometricsPrompt/BiometricsPrompt.js:168]: not needed anymore

File: src/screens/BiometricsPrompt/BiometricsPrompt.js [hunk 3]
```diff
@@ -39,15 +39,17 @@ import Toast from 'components/Toast';
...
 // types
 import type { Dispatch, RootReducerState } from 'reducers/rootReducer';
-
+import type { Theme } from 'models/Theme';
 
 type Props = {
   navigation: NavigationScreenProp<*>,
   registerWallet: (setBiometrics: boolean, themeToStore: string) => void,
   themeType: string,
+  theme: Theme,
```
Comment (graszka22) [inline src/screens/BiometricsPrompt/BiometricsPrompt.js:52]: not needed

**Pull request merged**